multicamera_acquisition/tests/visualization/conftest.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--rungui", action="store_true", default=False, help="run gui tests"
+    )
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "gui: mark test as slow to run")
```

Comment:
Any reason to not just add this to the main pytest.ini file? Should the pytest.ini file be at the python module level instead of the repo level, and that's why you had to put this here?